### PR TITLE
Use https for freedesktop.org

### PR DESCRIPTION
**Pull request by mbiebl**
`grep -l -r http:// | xargs sed -E -i s'#http://(.*).freedesktop.org#https://\1.freedesktop.org#'`

### CODE REVIEW

File: man/os-release.xml [hunk 1]
```diff
@@ -354,7 +354,7 @@
           <term><varname>LOGO=</varname></term>
 
           <listitem><para>A string, specifying the name of an icon as defined by <ulink
-          url="http://standards.freedesktop.org/icon-theme-spec/latest">freedesktop.org Icon Theme
+          url="https://standards.freedesktop.org/icon-theme-spec/latest".freedesktop.org Icon Theme
```
Comment (yuwata) [inline man/os-release.xml]: This breaks the tag...

Comment (mbiebl) [reply man/os-release.xml]: Thanks, fixed.

**Review by yuwata — APPROVED** [review]

**Comment by medhefgo** [comment]
Why only for freedesktop.org? This should really be done for all urls that have https…

**Comment by medhefgo** [comment]
I thought we did away with license boiler plate, but I guess that was forgotten for shell scripts?

**Comment by mbiebl** [comment]
@medhefgo I was too lazy to go through all http refs and just picked the low-hanging fruit.

If you want to see the other urls changed, please send a follow-up PR!

**Pull request merged**